run_bokchoy_tests.sh

```diff
@@ -0,0 +1,36 @@
+i#!/usr/bin/env bash
+
+# this stops the django servers
+stopServers() {
+    kill $(ps aux | grep "[m]anage.py" | awk '{print $2}')
+}
+
+echo "Stop any already running servers..."
+stopServers
+
+echo "Starting Notifications Server..."
+./manage.py runserver --settings=testserver.settings --noreload &
+
+
+echo "Waiting for testserver to fully start up..."
+
+until $(curl --output /dev/null --silent --head --fail http://127.0.0.1:8000); do
+    printf '.'
+    sleep 2
+done
+
+echo "Running acceptance tests..."
+python testserver/test/acceptance/test_login.py
```

Comment:
what should we do here when we have a whole test suite, is there a way to trigger the execution of all of them? Do we have to do some bash shell magic to do a 'find...' (all test_*.py) and iterate over the list?